css/interactive-guides/pod.scss

```diff
@@ -10,11 +10,11 @@
  *******************************************************************************/
 
 .podContainer {
-  height: 100%;
+  height: 200px;
```

Comment:
We would need to configure the pod height via json as a percentage and then calculate for the rest of the containers. This is doable but the pod will stay constant per guide. If you do more than 20% for pod it would look very big to fit in the other containers

Reply:
I go ahead and remove the height in pod.scss. 
The browser container overflows is something I still working on.